Source/Management.Web/App.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import React from 'react';
+import { Layout } from './layouts/Layout';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+
+import './App.scss';
+import { EventHandlersOverview } from './eventHandlers/EventHandlersOverview';
+import { Dashboard } from './home/Dashboard';
+import { Metrics } from './metrics/Metrics';
+
+
+export const App = () => {
+    return (
+        <>
+            <Router>
+                <Layout />
+                <div className="content">
+                    
+                <Route exact path="/">
+                        <Dashboard/>
+                    </Route>
+                    <Route exact path="/allmetrics">
```

Comment:
If were going to have more metrics pages, they could be under a sub-path maybe? Like `/metrics/some`?
```suggestion
                    <Route exact path="/metrics">
```

Reply:
:) To be honest, this ended up being «allmetrics» because I didn’t have a better name for it since the actual metrics already exists on /metrics.  Open for namingsuggestions. The UI could provide filtering if we want to.

Reply:
The path you suggest can’t be used - without us changing the /metrics for the Prometheus endpoint.